geocity/apps/submissions/forms.py

```diff
@@ -1105,7 +1105,23 @@ def _notify_author(self, submission):
             if submission.administrative_entity.expeditor_email
             else settings.DEFAULT_FROM_EMAIL
         )
-
+        is_awaiting_supplement = (
+            submission.status == models.Submission.STATUS_AWAITING_SUPPLEMENT
+        )
+        if is_awaiting_supplement:
```

Comment:
`if submission.status == models.Submission.STATUS_AWAITING_SUPPLEMENT:`

Remove variable that is used only once ?